probe/main.go

```diff
@@ -25,10 +25,11 @@ var version = "dev" // set at build time
 
 func main() {
 	var (
+		publish            = flag.String("publish", fmt.Sprintf("http://localhost:%d", xfer.AppPort), "URL to publish reports")
```

Comment:
Yep, this and the resolver is coming back in another commit, I just wanted a clean foundation to build up on.